layer/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine
+LABEL org.opencontainers.image.source "https://github.com/customink/crypteia"
```

Comment:
This is pretty neat... GitHub should use this label to associate the package with this repo.